<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="24" passed="18" failed="6" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-01-06T22:08:36 IST" name="Suite" finished-at="2023-01-06T22:09:03 IST" duration-ms="27362">
    <groups>
    </groups>
    <test started-at="2023-01-06T22:08:36 IST" name="Test" finished-at="2023-01-06T22:09:03 IST" duration-ms="27362">
      <class name="com.employeeapi.Test.TC_04_PUT_EMPLOYEE">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.employeeapi.Test.TC_04_PUT_EMPLOYEE@2bbaf4f0]" started-at="2023-01-06T22:08:54 IST" name="setUp" finished-at="2023-01-06T22:08:54 IST" duration-ms="18" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="updateEmployee()[pri:0, instance:com.employeeapi.Test.TC_04_PUT_EMPLOYEE@2bbaf4f0]" started-at="2023-01-06T22:08:54 IST" name="updateEmployee" finished-at="2023-01-06T22:09:00 IST" duration-ms="5885" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateEmployee -->
        <test-method signature="checkContentType()[pri:0, instance:com.employeeapi.Test.TC_04_PUT_EMPLOYEE@2bbaf4f0]" started-at="2023-01-06T22:09:00 IST" name="checkContentType" finished-at="2023-01-06T22:09:00 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkContentType -->
        <test-method signature="checkResposebody()[pri:0, instance:com.employeeapi.Test.TC_04_PUT_EMPLOYEE@2bbaf4f0]" started-at="2023-01-06T22:09:00 IST" name="checkResposebody" finished-at="2023-01-06T22:09:00 IST" duration-ms="7" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkResposebody -->
        <test-method signature="checkStatuscode()[pri:0, instance:com.employeeapi.Test.TC_04_PUT_EMPLOYEE@2bbaf4f0]" started-at="2023-01-06T22:09:00 IST" name="checkStatuscode" finished-at="2023-01-06T22:09:00 IST" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkStatuscode -->
        <test-method signature="checkStatusline()[pri:0, instance:com.employeeapi.Test.TC_04_PUT_EMPLOYEE@2bbaf4f0]" started-at="2023-01-06T22:09:00 IST" name="checkStatusline" finished-at="2023-01-06T22:09:00 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkStatusline -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.employeeapi.Test.TC_04_PUT_EMPLOYEE@2bbaf4f0]" started-at="2023-01-06T22:09:00 IST" name="teardown" finished-at="2023-01-06T22:09:00 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.employeeapi.Test.TC_04_PUT_EMPLOYEE -->
      <class name="com.employeeapi.Test.TC_05_DELETE_EmployeeRecord">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.employeeapi.Test.TC_05_DELETE_EmployeeRecord@17776a8]" started-at="2023-01-06T22:08:36 IST" name="setUp" finished-at="2023-01-06T22:08:36 IST" duration-ms="62" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="deleteEmployeeRecords()[pri:0, instance:com.employeeapi.Test.TC_05_DELETE_EmployeeRecord@17776a8]" started-at="2023-01-06T22:08:36 IST" name="deleteEmployeeRecords" finished-at="2023-01-06T22:08:44 IST" duration-ms="8132" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteEmployeeRecords -->
        <test-method signature="checkContentEncoding()[pri:0, instance:com.employeeapi.Test.TC_05_DELETE_EmployeeRecord@17776a8]" started-at="2023-01-06T22:08:44 IST" name="checkContentEncoding" finished-at="2023-01-06T22:08:44 IST" duration-ms="18" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkContentEncoding -->
        <test-method signature="checkResponseBody()[pri:0, instance:com.employeeapi.Test.TC_05_DELETE_EmployeeRecord@17776a8]" started-at="2023-01-06T22:08:44 IST" name="checkResponseBody" finished-at="2023-01-06T22:08:44 IST" duration-ms="25" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:751)
at org.testng.Assert.assertEquals(Assert.java:761)
at com.employeeapi.Test.TC_05_DELETE_EmployeeRecord.checkResponseBody(TC_05_DELETE_EmployeeRecord.java:41)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkResponseBody -->
        <test-method signature="checkStatuscode()[pri:0, instance:com.employeeapi.Test.TC_05_DELETE_EmployeeRecord@17776a8]" started-at="2023-01-06T22:08:44 IST" name="checkStatuscode" finished-at="2023-01-06T22:08:44 IST" duration-ms="8" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [429]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [429]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:839)
at org.testng.Assert.assertEquals(Assert.java:849)
at com.employeeapi.Test.TC_05_DELETE_EmployeeRecord.checkStatuscode(TC_05_DELETE_EmployeeRecord.java:49)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkStatuscode -->
        <test-method signature="checkserverType()[pri:0, instance:com.employeeapi.Test.TC_05_DELETE_EmployeeRecord@17776a8]" started-at="2023-01-06T22:08:45 IST" name="checkserverType" finished-at="2023-01-06T22:08:45 IST" duration-ms="2" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [nginx/1.14.1] but found [Apache]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [nginx/1.14.1] but found [Apache]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.employeeapi.Test.TC_05_DELETE_EmployeeRecord.checkserverType(TC_05_DELETE_EmployeeRecord.java:61)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkserverType -->
        <test-method signature="checkstatusline()[pri:0, instance:com.employeeapi.Test.TC_05_DELETE_EmployeeRecord@17776a8]" started-at="2023-01-06T22:08:45 IST" name="checkstatusline" finished-at="2023-01-06T22:08:45 IST" duration-ms="1" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [HTTP/1.1 200 OK] but found [HTTP/1.1 429 Too Many Requests]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [HTTP/1.1 200 OK] but found [HTTP/1.1 429 Too Many Requests]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.employeeapi.Test.TC_05_DELETE_EmployeeRecord.checkstatusline(TC_05_DELETE_EmployeeRecord.java:55)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkstatusline -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.employeeapi.Test.TC_05_DELETE_EmployeeRecord@17776a8]" started-at="2023-01-06T22:08:45 IST" name="teardown" finished-at="2023-01-06T22:08:45 IST" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.employeeapi.Test.TC_05_DELETE_EmployeeRecord -->
      <class name="com.employeeapi.Test.TC_02_GET_Sinfgle_EmployeeRecord">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.employeeapi.Test.TC_02_GET_Sinfgle_EmployeeRecord@2d127a61]" started-at="2023-01-06T22:08:50 IST" name="setUp" finished-at="2023-01-06T22:08:51 IST" duration-ms="8" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="getAllEmployees()[pri:0, instance:com.employeeapi.Test.TC_02_GET_Sinfgle_EmployeeRecord@2d127a61]" started-at="2023-01-06T22:08:51 IST" name="getAllEmployees" finished-at="2023-01-06T22:08:54 IST" duration-ms="3195" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getAllEmployees -->
        <test-method signature="checkContentType()[pri:0, instance:com.employeeapi.Test.TC_02_GET_Sinfgle_EmployeeRecord@2d127a61]" started-at="2023-01-06T22:08:54 IST" name="checkContentType" finished-at="2023-01-06T22:08:54 IST" duration-ms="3" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [txt/html; charset=UTF-8] but found [application/json]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [txt/html; charset=UTF-8] but found [application/json]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.employeeapi.Test.TC_02_GET_Sinfgle_EmployeeRecord.checkContentType(TC_02_GET_Sinfgle_EmployeeRecord.java:66)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkContentType -->
        <test-method signature="checkContentlength()[pri:0, instance:com.employeeapi.Test.TC_02_GET_Sinfgle_EmployeeRecord@2d127a61]" started-at="2023-01-06T22:08:54 IST" name="checkContentlength" finished-at="2023-01-06T22:08:54 IST" duration-ms="4" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkContentlength -->
        <test-method signature="checkResponsebody()[pri:0, instance:com.employeeapi.Test.TC_02_GET_Sinfgle_EmployeeRecord@2d127a61]" started-at="2023-01-06T22:08:54 IST" name="checkResponsebody" finished-at="2023-01-06T22:08:54 IST" duration-ms="13" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkResponsebody -->
        <test-method signature="checkResponsetime()[pri:0, instance:com.employeeapi.Test.TC_02_GET_Sinfgle_EmployeeRecord@2d127a61]" started-at="2023-01-06T22:08:54 IST" name="checkResponsetime" finished-at="2023-01-06T22:08:54 IST" duration-ms="7" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkResponsetime -->
        <test-method signature="checkServertype()[pri:0, instance:com.employeeapi.Test.TC_02_GET_Sinfgle_EmployeeRecord@2d127a61]" started-at="2023-01-06T22:08:54 IST" name="checkServertype" finished-at="2023-01-06T22:08:54 IST" duration-ms="3" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [nginx/1.14.1] but found [nginx/1.21.6]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [nginx/1.14.1] but found [nginx/1.21.6]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.employeeapi.Test.TC_02_GET_Sinfgle_EmployeeRecord.checkServertype(TC_02_GET_Sinfgle_EmployeeRecord.java:74)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkServertype -->
        <test-method signature="checkStatusCode()[pri:0, instance:com.employeeapi.Test.TC_02_GET_Sinfgle_EmployeeRecord@2d127a61]" started-at="2023-01-06T22:08:54 IST" name="checkStatusCode" finished-at="2023-01-06T22:08:54 IST" duration-ms="13" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkStatusCode -->
        <test-method signature="checkStatusline()[pri:0, instance:com.employeeapi.Test.TC_02_GET_Sinfgle_EmployeeRecord@2d127a61]" started-at="2023-01-06T22:08:54 IST" name="checkStatusline" finished-at="2023-01-06T22:08:54 IST" duration-ms="4" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkStatusline -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.employeeapi.Test.TC_02_GET_Sinfgle_EmployeeRecord@2d127a61]" started-at="2023-01-06T22:08:54 IST" name="teardown" finished-at="2023-01-06T22:08:54 IST" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.employeeapi.Test.TC_02_GET_Sinfgle_EmployeeRecord -->
      <class name="com.employeeapi.Test.TC_03_POST_EmployeeData">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.employeeapi.Test.TC_03_POST_EmployeeData@69a10787]" started-at="2023-01-06T22:08:45 IST" name="setUp" finished-at="2023-01-06T22:08:45 IST" duration-ms="13" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="createEmployee()[pri:0, instance:com.employeeapi.Test.TC_03_POST_EmployeeData@69a10787]" started-at="2023-01-06T22:08:45 IST" name="createEmployee" finished-at="2023-01-06T22:08:50 IST" duration-ms="5907" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createEmployee -->
        <test-method signature="checkContentType()[pri:0, instance:com.employeeapi.Test.TC_03_POST_EmployeeData@69a10787]" started-at="2023-01-06T22:08:50 IST" name="checkContentType" finished-at="2023-01-06T22:08:50 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkContentType -->
        <test-method signature="checkResposebody()[pri:0, instance:com.employeeapi.Test.TC_03_POST_EmployeeData@69a10787]" started-at="2023-01-06T22:08:50 IST" name="checkResposebody" finished-at="2023-01-06T22:08:50 IST" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkResposebody -->
        <test-method signature="checkStatuscode()[pri:0, instance:com.employeeapi.Test.TC_03_POST_EmployeeData@69a10787]" started-at="2023-01-06T22:08:50 IST" name="checkStatuscode" finished-at="2023-01-06T22:08:50 IST" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkStatuscode -->
        <test-method signature="checkStatusline()[pri:0, instance:com.employeeapi.Test.TC_03_POST_EmployeeData@69a10787]" started-at="2023-01-06T22:08:50 IST" name="checkStatusline" finished-at="2023-01-06T22:08:50 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkStatusline -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.employeeapi.Test.TC_03_POST_EmployeeData@69a10787]" started-at="2023-01-06T22:08:50 IST" name="teardown" finished-at="2023-01-06T22:08:50 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.employeeapi.Test.TC_03_POST_EmployeeData -->
      <class name="com.employeeapi.Test.TC_01_GET_All_Employees">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.employeeapi.Test.TC_01_GET_All_Employees@11c20519]" started-at="2023-01-06T22:09:00 IST" name="setUp" finished-at="2023-01-06T22:09:00 IST" duration-ms="12" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="getAllEmployees()[pri:0, instance:com.employeeapi.Test.TC_01_GET_All_Employees@11c20519]" started-at="2023-01-06T22:09:00 IST" name="getAllEmployees" finished-at="2023-01-06T22:09:03 IST" duration-ms="3173" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getAllEmployees -->
        <test-method signature="checkResponsebody()[pri:0, instance:com.employeeapi.Test.TC_01_GET_All_Employees@11c20519]" started-at="2023-01-06T22:09:03 IST" name="checkResponsebody" finished-at="2023-01-06T22:09:03 IST" duration-ms="9" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkResponsebody -->
        <test-method signature="checkResponsetime()[pri:0, instance:com.employeeapi.Test.TC_01_GET_All_Employees@11c20519]" started-at="2023-01-06T22:09:03 IST" name="checkResponsetime" finished-at="2023-01-06T22:09:03 IST" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkResponsetime -->
        <test-method signature="checkStatusCode()[pri:0, instance:com.employeeapi.Test.TC_01_GET_All_Employees@11c20519]" started-at="2023-01-06T22:09:03 IST" name="checkStatusCode" finished-at="2023-01-06T22:09:03 IST" duration-ms="4" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkStatusCode -->
        <test-method signature="checkStatusline()[pri:0, instance:com.employeeapi.Test.TC_01_GET_All_Employees@11c20519]" started-at="2023-01-06T22:09:03 IST" name="checkStatusline" finished-at="2023-01-06T22:09:03 IST" duration-ms="9" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkStatusline -->
      </class> <!-- com.employeeapi.Test.TC_01_GET_All_Employees -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
